tests/test_atomic_models.py

```diff
@@ -60,6 +62,48 @@ def test_read_atomic_model_from_cif(self):
         model = read_atomic_model(path)
         assert model.__class__ is gemmi.Model
 
+    def test_extract_gemmi_atoms(self):
+        """Check that extraction retrieves correct no. of chains."""
+        pdb_filename = "2dhb.pdb"
+        path = os.path.join(DATA, pdb_filename)
+        os.system(f"wget https://files.rcsb.org/download/{pdb_filename} -P {DATA}")
```

Comment:
Would it make sense to `os.delete` the file at the end of the test?

Reply:
Good catch! Yes this is definitely a good idea